src/gp_engine.hpp

```diff
@@ -67,13 +78,15 @@ class GPEngine {
  private:
   static constexpr double min_branch_length_ = 1e-6;
   static constexpr double max_branch_length_ = 3.;
+  static constexpr double min_log_branch_length_ = -13.9;
```

Comment:
It doesn't seem like we use the non-log versions anymore, so let's discard them.

Reply:
Done. `min_branch_length_` and `max_branch_length_` are removed.